packages/components/select/src/useSelect.ts

```diff
@@ -91,6 +91,18 @@ export const useSelect = (props: ISelectProps, emit) => {
     handleScroll: () => void
   } | null>(null)
 
+  const {
+    isComposing,
+    handleCompositionStart,
+    handleCompositionUpdate,
+    handleCompositionEnd,
+  } = useComposition({
+    afterComposition(event) {
+      const text = (event.target as HTMLInputElement)?.value
+      handleQueryChange(text)
+    },
```

Comment:
The function handled by `afterComposition` here should be `onInput` like `select-v2`.